great_expectations_cloud/agent/actions/draft_datasource_config_action.py

```diff
@@ -69,9 +69,12 @@ def check_draft_datasource_config(
             created_resources=[],
         )
 
-    def _get_table_names(self, datasource: Datasource) -> list[str]:
+    def _get_table_names(self, datasource: Datasource, *, include_views: bool) -> list[str]:
```

Comment:
Rename to `_get_table_and_view_names` and remove the boolean flag. We don't have a need for the conditional and it makes the API less clear.